### ISSUE TITLE
Explorer not showing all registered transcoder nodes

### ISSUE THREAD

**Issue by dob** [livepeer/livepeer-monorepo#434]
**Describe the bug (required)**
It looks like our explorer isn't showing all the transcoders for some reason. It lists only 23 active. After doing a comparison with Scout, I noticed these particular nodes are missing (in my view). Note also that Scout is missing some nodes, but it's not on us to debug their index:

Explorer is missing:
0xE9E284277648fcdb09B8EfC1832c73c09b5Ecf59
0xFb9849b0B53F66B747Bfa47396964a3FA22400A0

Scout is missing:
0x9c10672cee058fd658103d90872fe431bb6c0afa
0x9e14de5cc84e437518c2c303c88570fe13f0ad15
0xd84781e1a9b74d71ea76cda8bb9f30893bfd00d1

**Expected behavior (required)**
All registered nodes should be shown.

**To Reproduce (required)**
1. Go to https://explorer.livepeer.org
2. View the list of transcoders. 
3. Observe that the above mentioned nodes are missing.

**Device (required):**
 - Device: Macbook Pro, OS X, Chrome (and Brave)
 
**Additional Context:**

Adam had mentioned that the Graph node was updated recently. This may have introduced an issue.

**Comment by adamsoffer** [comment]
Eli and I upgraded the Graph Node to the latest version yesterday, which seems to now paginate graphql query results (100 results per page). Currently, the explorer queries for all transcoders and then filters out those that are registered and active client-side. Since the latest Graph Node now paginates results, that means we're excluding some from the list. I've confirmed the two missing transcoders are on the second page of results. 

I think I simple fix would be to include a filter on our transcoders query. ie:
```
{
  transcoders(where:{status: "Registered"}) {
     ...
  }
}
```
